app/src/main/res/layout/log_in_fragment.xml

```diff
@@ -0,0 +1,93 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <!-- TODO: Update blank fragment layout -->
+
+    <ImageView
+        android:id="@+id/imageView2"
+        android:layout_width="325dp"
+        android:layout_height="330dp"
+        android:src="@mipmap/dog_fauna_foreground"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <TextView
+        android:id="@+id/textView"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+
+        android:layout_marginTop="272dp"
+
+        android:text="@string/fauna"
+        android:textSize="35sp"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0.498"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+
+    <com.google.android.material.textfield.TextInputLayout
+        android:id="@+id/textInputLayout2"
+        android:layout_width="220dp"
+        android:layout_height="32dp"
+        android:layout_marginTop="16dp"
```

Comment:
Define some standard dimens in a values file, and use them in your layouts. 
Eg:

`<resources>
    <dimen name="keyline_0">2dp</dimen>
    <dimen name="keyline_1">4dp</dimen>
    <dimen name="keyline_2">8dp</dimen>
    <dimen name="keyline_4">16dp</dimen>
    <dimen name="keyline_5">24dp</dimen>

    <dimen name="radius_0">30dp</dimen>

    <dimen name="stroke_0">2dp</dimen>
</resources>`

Reply:
Didn't know about it- will change soon